src/components/Search/index.js

```diff
@@ -1,16 +1,88 @@
-import React from 'react';
-import { Input } from '@material-ui/core';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
+import { fade, makeStyles } from '@material-ui/core/styles';
+import SearchIcon from '@material-ui/icons/Search';
+import InputBase from '@material-ui/core/InputBase';
+
+const useStyles = makeStyles(theme => ({
+  search: {
```

Comment:
This feels like a smell to me from the perspective of if we're creating a lot of custom CSS for a Material UI Component, then we're working against Material's design system. IMO, might be worth dropping?

Reply:
Actually this custom CSS was copied directly from Material UI from their demo of a Search bar here: https://material-ui.com/components/app-bar/

I think their demo search with the icon looks kind of nice, though:
![image](https://user-images.githubusercontent.com/4512699/93170539-463f1a00-f6dc-11ea-85ef-19a06badb0ef.png)